node/as/http.js

```diff
@@ -311,7 +311,9 @@ TChannelHTTP.prototype.forwardToHTTP = function forwardToHTTP(tchannel, options,
 
 TChannelHTTP.prototype._forwardToLBPool = function _forwardToLBPool(options, inreq, outres, callback) {
     var self = this;
-
+    options = extend(options, {
```

Comment:
Extend is slow, especially on an every-request method like this; we've abolished all (most?) usage of it from production code paths.  Please instead:
```
if (!options) { options = {}; }
options.encoding = null;
```

Mutating the passed options directly should be okay, but if you'd really rather not look at existing tchannel options handling code paths for inspiration.